cs-deployment.yaml

```diff
@@ -27,17 +27,17 @@ spec:
     metadata:
       labels:
         app: codescene
-        version: 3
+        version: "3"
     spec:
       containers:
       - image: empear/ubuntu-onprem
         env:
         - name: CODESCENE_DB_PATH
-          value: "/opt/codescene/codescene"
+          value: "/data/codescene/codescene"
         - name: CODESCENE_ANALYSIS_RESULTS_ROOT
-          value: "/opt/codescene/analysis"
+          value: "/data/codescene/analysis"
         - name: CODESCENE_CLOSED_REPOSITORIES_ROOT
```

Comment:
This is a typo (probably) copy-pasted from the earlier version of https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl.
Should be `CODESCENE_CLONED_REPOSITORIES_ROOT`.

Reply:
Indeed it is a typo. Good catch.